Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -17,8 +17,16 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
     _ChatMessageActionsVCDelegate,
     ChatMessageContentViewDelegate,
     UICollectionViewDelegate,
-    UICollectionViewDataSource,
+    ChatMessageListCollectionViewDataSource,
     GalleryContentViewDelegate {
+    open var overlayDateFormatter: DateFormatter = {
+        let df = DateFormatter()
+        df.dateStyle = .short
+        df.timeStyle = .none
+        df.locale = .autoupdatingCurrent
+        return df
+    }()
+    
```

Comment:
Can you please add documentation and ideally also move it somewhere else in the file? Like this it looks like this is the most important thing in the controller :D